tests/unit/fixtures/alternative-structure.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+// @phpcsSniff InpsydeTemplates.Formatting.AlternativeControlStructure
+
+const FLAGS = [
+    'YES',
+    'NO',
+    'MAYBE',
+];
+
+$flag = FLAGS[rand(0, 2)];
+
+if ($flag === 'MAYBE') {
+    echo 'maybe';
+
+    while ($flag !== 'YES') {
+        $flag = 'YES';
+    }
+} elseif ($flag === 'NO') {
+    echo 'no';
+} else if ($flag === 'YES') {
+    echo 'yes';
+} else {
+    echo 'Non Empty value';
+}
+
+if ($flag === 'MAYBE') :
+    echo 'maybe';
+
+    while ($flag !== 'YES') {
+        $flag = 'YES';
+    }
+elseif ($flag === 'NO') :
+    echo 'no';
+else :
+    echo 'Non Empty value';
+endif;
+
+
+$arrayOfFlags = [];
+for ($i = 1; $i <= 10; $i++) {
+    $arrayOfFlags[] = FLAGS[rand(0, 2)];
+}
+
+foreach ($arrayOfFlags as &$item) {
+    $item = false;
+}
+unset($item);
+
+switch ($flag) {
+    case 'YES':
+        echo 'It is true';
+        break;
+    case 'NO':
+        echo 'It is false';
+        break;
+}
+
+?>
+
+<?php if ($flag === 'MAYBE') { // @phpcsWarningOnThisLine ?>
+    <div>Maybe.</div>
+    <?php while ($flag !== 'YES') {
+        $flag = 'YES';
+    }
+} elseif ($flag === 'NO') { // @phpcsWarningOnThisLine
+    echo 'no';
```

Comment:
So far, we only have a nesting level of one, but the sniff should also target nested control structures with inline HTML that should be using alternative syntax. I'd like to replace the `NO` case with some nested code that does get flagged:

```suggestion
    while ($flag !== 'YES') { // @phpcsWarningOnThisLine
        $flag = 'YES';
        ?>
        <div>No. Yes.</div>
        <?php
    }
```